src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java

```diff
@@ -80,11 +80,12 @@ public void should_throw_exception_when_zip_file_has_mismatching_pdf() throws IO
     }
 
     private ZipFileProcessor getZipFileProcessor(String zipFileName) throws IOException, URISyntaxException {
-        ZipFileProcessor processor = new ZipFileProcessor("container", zipFileName);
+        String container = "container";
+        ZipFileProcessor processor = new ZipFileProcessor(container, zipFileName);
```

Comment:
can `container` be inlined?

Reply:
It can but I don't like it, because it's used more than once in that method, so I'd rather declare it separately.

Reply:
oops, didn't notice it's used twice